setup.py

```diff
@@ -48,7 +48,7 @@ def read_readme():
     package_dir={"": "src"},
     python_requires=">=3.6",
     install_requires=[
-        "globus-sdk==3.3.1",
+        "globus-sdk==3.5.0",
```

Comment:
I'm surprised that this is not considered a conflict. I'm not 100% certain what `git blame` would show on `setup.py` after a merge.